Backend/dev/migrations/rider-app/1116-rating.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE atlas_app.rating (
+id character(36) NOT NULL,
+ride_id character varying(36) NOT NULL,
+rating_value bigint,
+feedback_details character varying(255),
+rider_id character varying(36) NOT NULL,
```

Comment:
ride_id and rider_id should have character(36)
Should rating_value be bigint ? What is its range